tests/wpt/mozilla/tests/mozilla/text-overflow-ellipsis-stacking-context.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<link rel="match" href="text-overflow-ellipsis-stacking-context-ref.html">
+<style>
+  .woo {
+      min-width: 1px;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      box-sizing: border-box;
+      padding-left: 0.25em;
+      display: inline-block;
+  }
+
+  .animate {
+      transform: translateX(1px);
+      display: inline-block;
+  }
```

Comment:
There’s quite a bit going on here. Is all of it necessary to reproduce the panic without the Rust code change, or could this test be reduced?

Reply:
This is actually reduced as much as possible. Any change to the CSS rules or properties causes the panic to disappear.

Reply:
Nevermind, I was able to make it more clear and renamed the classes as well.